src/definition.ts

```diff
@@ -0,0 +1,11 @@
+import { providers } from "ethers";
```

Comment:
what about 

- https://stackoverflow.com/questions/65504958/web3-js-extending-the-window-interface-type-definitions
- https://stackoverflow.com/questions/65504958/web3-js-extending-the-window-interface-type-definitions#comment120290255_67665205
- https://github.com/MetaMask/providers/tree/main/src